main.go

```diff
@@ -65,6 +66,10 @@ func run(c *cli.Context) error {
 	log.Printf("Main: Start: %v", start)
 	var G, fullGraph *Graph
 
+	if c.IsSet("file") && c.IsSet("puppet") {
+		log.Fatal("the --file and --puppet parameters cannot be used together")
```

Comment:
Good, but this should be a parse error instead...
Try using:

		log.Printf("blah blah...")
		return cli.NewExitError("", 1)

higher up, maybe right after the noop := part